README.md

```diff
@@ -1 +1,70 @@
-# maths-magicians
\ No newline at end of file
+# Getting Started with Create React App
```

Comment:
- [ ] Kindly ensure all the main sections from the [readme template](https://github.com/microverseinc/readme-template/blob/master/README.md) are present, also remove the unnecessary parts.